### PR TITLE
Add genome attributes design

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: design/genome_attributes_filtering.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      * No need for this currently AFAWK
+    * Filter strategies
+      * prefix search
+      * full text search
+        * requires ArangoSearch View or inverted index
+      * substring search
+        * cannot use index without ArangoSearch View or (maybe?) inverted index
+          * TBD - will an inverted index work?
+        * Out of scope for v1
+  * Enum
```
Comment (Tianhao-Gu) [inline design/genome_attributes_filtering.md:26]: enum includes both list and dictionary type?

Comment (MrCreosote) [reply design/genome_attributes_filtering.md:26]: Not sure what you mean - an enum is a set of strings

Comment (MrCreosote) [reply design/genome_attributes_filtering.md:26]: I still have no idea what you mean - there aren't any dictionary values for enums

**Review by dauglyon — APPROVED** [review]

File: design/genome_attributes_filtering.md (hunk 1 above)
Comment (MrCreosote) [reply design/genome_attributes_filtering.md:26]: I don't think there is a dictionary column type. Which column were you thinking of?

Comment (Tianhao-Gu) [reply design/genome_attributes_filtering.md:26]: currently I only see `_upas`.

Comment (MrCreosote) [reply design/genome_attributes_filtering.md:26]: Those aren't visible to the user, they're for the service only

Comment (MrCreosote) [reply design/genome_attributes_filtering.md:26]: https://github.com/kbase/collections/blob/main/src/service/data_products/genome_attributes.py#L212

Comment (Tianhao-Gu) [reply design/genome_attributes_filtering.md:26]: 👍

**Review by Tianhao-Gu — APPROVED** [review]
👍

**Pull request merged**